etfl/core/genes.py

```diff
@@ -77,7 +78,27 @@ def transcribed_by(self):
     @transcribed_by.setter
     def transcribed_by(self,value):
         # TODO: Make this a setter that rewrites the adequate constraints
-        raise NotImplementedError()
+        if value != self._transcribed_by:
+            if self.model is None:
+                # Easy
+                self._transcribed_by = value
+            else:
+                # We need to make the model change the corresponding cstr
+                mod_id = self.id + '_transcription'
+                Cstr = self.model.get_constraints_of_type(SynthesisConstraint)
```

Comment:
please respect the conventions and use Uppercas names only for classes. variables are lowercase most of the time.